packages/features/borrow/src/borrow-fixed/hooks/use-fixed-liquidity-pools-table.tsx

```diff
@@ -0,0 +1,106 @@
+import { useContext } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { BorrowFixedContext } from '../../borrow-fixed/borrow-fixed';
+import { useFCashMarket } from '@notional-finance/notionable-hooks';
+import { DisplayCell, DataTableColumn } from '@notional-finance/mui';
+import { getNowSeconds } from '@notional-finance/util';
+import { RATE_PRECISION } from '@notional-finance/util';
+import {
+  getDateString,
+  formatNumber,
+  formatNumberAsPercent,
+} from '@notional-finance/helpers';
+
+export const useFixedLiquidityPoolsTable = (
+  selectedDepositToken: string | undefined
+) => {
+  const {
+    state: { deposit },
+  } = useContext(BorrowFixedContext);
+  const fCashMarket = useFCashMarket(deposit?.currencyId);
+  let tableData: any[] = [];
+  let tableColumns: DataTableColumn[] | [] = [];
+  if (selectedDepositToken) {
+    tableColumns = [
+      {
+        Header: (
+          <FormattedMessage
+            defaultMessage={'Maturity'}
+            description={'Maturity header'}
+          />
+        ),
+        Cell: DisplayCell,
+        accessor: 'maturity',
+        textAlign: 'left',
+      },
+      {
+        Header: `${selectedDepositToken}`,
+        Cell: DisplayCell,
+        accessor: 'valueOfCash',
+        textAlign: 'left',
+      },
+      {
+        Header: `F${selectedDepositToken}`,
+        Cell: DisplayCell,
+        accessor: 'valueOfFCash',
+        textAlign: 'left',
+      },
+      {
+        Header: (
+          <FormattedMessage
+            defaultMessage={'F{selectedDepositToken} PRICE'}
+            values={{
+              selectedDepositToken,
+            }}
+          />
+        ),
+        Cell: DisplayCell,
+        accessor: 'price',
+        textAlign: 'right',
+      },
+      {
+        Header: (
+          <FormattedMessage
+            defaultMessage={'Interest Rate'}
+            description={'Interest Rate header'}
+          />
+        ),
+        Cell: DisplayCell,
+        displayFormatter: formatNumberAsPercent,
+        accessor: 'interestRate',
+        textAlign: 'right',
+      },
+    ];
+  }
+
+  if (fCashMarket) {
+    const {
+      poolParams: { perMarketCash, perMarketfCash },
+    } = fCashMarket;
+
+    tableData = perMarketfCash.map((data, index) => {
+      const interestRate = fCashMarket.getSpotInterestRate(data.token);
+      const currentPrice = interestRate
```

Comment:
If the interestRate === 0 then this will return false and have a price of 0. If there interest rate is zero the price should be 1.  Instead, change this to `interestRate !== undefined ? ... : ...`